yodeploy/hooks/django.py

```diff
@@ -43,7 +43,7 @@ def prepare_logfiles(self):
                 continue
             with ignoring(errno.EEXIST):
                 os.mkdir(os.path.dirname(logfile))
-            touch(logfile, self.log_user, self.log_group, 0o640)
+            touch(logfile, self.log_user, self.log_group, 0o660)
```

Comment:
What's the reason behind it?

Reply:
please, see attached ticket

Reply:
I don't have permission to see the ticket. :disappointed:

Reply:
@deonbredenhann We want to enable people to run `./manage.py shell` without `sudo` :)

Right now mere developers don't have access to write to the log file.

Reply:
👍 😄